.buildkite/tests.sh

```diff
@@ -51,6 +51,11 @@ sleep 1 # WSL2 requires a wait after to avoid "Text file busy".
 "$earthly" --version
 "$earthly" config global.local_registry_host 'tcp://127.0.0.1:8371'
 
+# Yes, there is a bug in the upstream YAML parser. Sorry about the jank here.
+"$earthly" config global.buildkit_additional_config "'[registry.\"docker.io\"]
+
+ mirrors = [\"registry-1.docker.io.mirror.corp.earthly.dev\"]'"
```

Comment:
This can be fixed here and in the YAML if https://github.com/go-yaml/yaml/issues/423 ever gets fixed.

Reply:
can you add a link to that github issue in the comment, so that way if we read the comment later, we can reference the github issue more easily and updated it if it's fixed?